src/ResourceManager/Sql/Commands.Sql/Failover Group/Cmdlet/AddAzureSqlDatabaseToFailoverGroup.cs

```diff
@@ -44,19 +54,11 @@ public class AddAzureSqlDatabaseToFailoverGroup : AzureSqlFailoverGroupCmdletBas
         /// </summary>
         [Parameter(Mandatory = true,
             ValueFromPipeline = true,
-            HelpMessage = "The Azure SQL Database to be added to the secondary server.")]
+            HelpMessage = "One or more Azure SQL Databases on the Failover Group's primary server to be added to the Failover Group.")]
         [ValidateNotNullOrEmpty]
         public List<AzureSqlDatabaseModel> Database { get; set; }
```

Comment:
The reason is that it makes it possible to do things like "Get-AzureRmSqlDatababase <...> | Add-AzureRmSqlDatabaseToFailoverGroup <...>" without the need to write any other PowerShell code. It needs to be a single command -- adding the databases one by one would take too long.

I think an argument could be made that we should also have a 'DatabaseName' parameter, but in my opinion that is redundant since this one already exists.